include/rtc/candidate.hpp

```diff
@@ -25,6 +25,15 @@
 
 namespace rtc {
 
+#if not USE_JUICE
+struct CandidateInfo {
+	string address;
+	int port;
+	string type;
+	string transportType;
```

Comment:
I'd be more practical if these were enums. You could still define an `operator<<` like for `PeerConnection::State` to print it as string transparently.